redos/7.3/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM centos:7 as prepare
+MAINTAINER Sergei Vorontsov "piligrim@rootnix.net"
+
+ENV REPO_URL=http://mirror.yandex.ru/redos/7.3/x86_64/os/ \
+    CHROOT_DIR="/chroot"
+
+RUN mkdir -p $CHROOT_DIR
+
+#Download any available meta package redos-release
+RUN for rpm in `curl -s $REPO_URL | grep 'redos-release' | sed 's/<.*>\(.*\)<\/.*>/\1/' | awk '{print $1}'`; \
+    do \
```

Comment:
Usually `do` goes on one level with `for`:

```
for i in $(ls /tmp/); do 
    ...
done
```

Reply:
I started to follow this recommendation, but ended up with some rewritting of this block. A think of taste, but lines now shorter. I have added myself into co-authors to let people blame me about my mistakes.